content/id/docs/concepts/configuration/resource-bin-packing.md

```diff
@@ -0,0 +1,214 @@
+---
+title: _Bin Packing_ Sumber Daya untuk Sumber Daya Tambahan
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="1.16" state="alpha" >}}
+
+_Kube-scheduler_ dapat dikonfigurasikan untuk mengaktifkan _bin packing_ sumber 
+daya bersama dengan sumber daya tambahan melalui fungsi prioritas 
+`RequestedToCapacityRatioResourceAllocation`. Fungsi-fungsi prioritas dapat digunakan 
+untuk menyempurnakan _kube-scheduler_ sesuai dengan kebutuhan.
```

Comment:
Is it worth explaining what “bin packing” means?

Reply:
Yes... but it may not totally correct.